nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementError
```

Comment:
- [ ] This should be "NotImplementedError", note the ("ed"). It would also be nice to provide some context for the class in which the method is not implemented.

  ```
  raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  ```